packages/fxa-auth-server/lib/oauth/client.js

```diff
@@ -9,7 +9,7 @@ const Joi = require('@hapi/joi');
 const OauthError = require('./error');
 const validators = require('./validators');
 const db = require('./db');
-const encrypt = require('fxa-shared/auth/encrypt');
+const encrypt = require('./encrypt');
```

Comment:
Maybe could have kept this?

Reply:
Oh yes that probably doesn't need to reverted; however, For simplicity sake, I am going to roll with a revert on fxa-auth-server/**/* files. It's the simplest approach for now. Hopefully these reverted changes are only in play for a short time anyways.